.github/workflows/nix-config-base.json

```
@@ -0,0 +1,27 @@
+{
+  "editions": [
+    {
+      "name": "vast",
+      "static": true,
+      "copy-package": true,
```

This needs a better name

```suggestion
      "upload-package-to-github": true,
```